src/api/public/apidocs-new/paths/person_login.yaml

```diff
@@ -0,0 +1,44 @@
+post:
+  summary: Perform changes on a registered person
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
```

Comment:
@danidoni the login parameter is already defined in `../parameters/login.yaml`

Reply:
Yes!